src/hotspot/share/runtime/os.hpp

```diff
@@ -508,7 +508,7 @@ class os: AllStatic {
   // Map memory to the file referred by fd. This function is slightly different from map_memory()
   // and is added to be used for implementation of -XX:AllocateHeapAt
   static char* map_memory_to_file(size_t size, int fd, MEMFLAGS flag = mtNone);
-  static char* map_memory_to_file_aligned(size_t size, size_t alignment, int fd, MEMFLAGS flag = mtNone);
+  static char* map_memory_to_file_aligned(size_t size, size_t alignment, int fd, MEMFLAGS flag);
```

Comment:
There are still a few mtNone usages in this file.

Reply:
`mtNone` as default value for optional arguments is removed from all function definitions.